pyqt-apps/siriushla/li_ap_mps/main.py

```diff
@@ -0,0 +1,533 @@
+            if title not in ['Water Temperature', 'Temperature']:
+                layout = self.setTitleLabel(lbl_item[1], 1, layout)
+            else:
+                layout = self.setTempHeader(lbl_item[1], layout)
+        return layout
+
+    def getParamAll(self, has_title, pv_name, wid_type):
+        if wid_type == 0 or pv_name in ['PPState7',
+                'Mod1State', 'GPS1', 'WaterState', 'K1PsState',
+                'IP1Warn', 'CCG1Warn', 'CCG1Alarm', 'PRG1Warn',
+                'WFS1', 'WFS2', 'WFS3', 'WFS4', 'WFS5', 'WFS6']:
+            return has_title
+        else:
+            return False
+
+    def dispayHiddenControls(self, pv_name, control, config):
+        if control:
+            return self.controlWidget(
+                pv_name, '',
+                config)
+        else:
+            return self.statusBox(pv_name, config)
+
+    def dispayAllControls(self, pv_name, control, config):
+        if control:
+            widget = QWidget()
+            hlay = QHBoxLayout()
+            if pv_name.find('WFS') != -1:
+                hlay.addWidget(
+                    self.setPvLbl(pv_name))
+            control_layout = self.controlBox(pv_name, '', config, control, 1)
+            widget.setLayout(control_layout)
+            hlay.addWidget(widget)
+            return hlay
+        else:
+            return self.statusBox(pv_name, config)
+
+    def displayGroup(self, pv_data, title, group_type):
+        ''' Display one MPS group '''
+        dg_glay = QGridLayout()
+        group = QGroupBox()
+        count = [1, 1]
+
+        dg_glay = self.getSingleTitle(title, dg_glay)
+        index = 0
+        for pv_name in pv_data.get('name'):
+            if title != 'Gate Valve':
+                if group_type == 0:
+                    dg_glay.addLayout(
+                        self.dispayHiddenControls(
+                            pv_name,
+                            self.getPVControl(pv_data.get('control'), index),
+                            self.getPVConfig(pv_data.get('config'), index)),
+                        count[0], count[1], 1, 1)
+                else:
+                    dg_glay.addLayout(
+                        self.dispayAllControls(
+                            pv_name,
+                            self.getPVControl(pv_data.get('control'), index),
+                            self.getPVConfig(pv_data.get('config'), index)),
+                        count[0], count[1], 1, 1)
+            else:
+                dg_glay.addLayout(
+                    self.gateValve(pv_name,
+                        self.getPVConfig(pv_data.get('config'), index)),
+                    count[0], count[1], 1, 1)
+            count = self.updateCount(count, title)
+            index += 1
+
+        group.setTitle(title)
+        group.setLayout(dg_glay)
+
+        return group
+
+    def displayTempGroup(self, pv_data, title):
+        ''' Display one temperature group'''
+        dtg_glay = QGridLayout()
```

Comment:
Use horizontal spacing zero here to align title with column content
`dtg_glay.setHorizontalSpacing(0)`